docs/installing.md

```diff
@@ -129,3 +124,74 @@ It is also recommended to have:
 
 Additional resources may be required depending on the workloads you plan to run.
 
+### Installation via .deb Package
+
+Add the Rancher Desktop repository and install Rancher Desktop with:
+
+```
+curl https://download.opensuse.org/repositories/isv:/Rancher:/stable/deb/Release.key | sudo apt-key add -
+sudo add-apt-repository 'deb https://download.opensuse.org/repositories/isv:/Rancher:/stable/deb/ ./'
+sudo apt update
+sudo apt install rancher-desktop
+```
+
+### Uninstalling .deb Package
+
+You can remove the package, repository and key with:
```

Comment:
I personally prefer the Oxford comma `, and`

Reply:
I prefer the oxford comma, too.